src/create-model.jl

```diff
@@ -902,8 +926,12 @@ function create_model(
     @timeit to "multi-year investment" begin
         @expression(
             model,
-            accumulate_capacity_simple[y ∈ Y, a ∈ Ai[y]],
-            sum(assets_investment[yi, a] for yi in Yi[a] if starting_year[(y, a)] ≤ yi ≤ y)
+            accumulate_capacity_simple_method[y ∈ Y, a ∈ investable_assets_using_simple_method[y]],
```

Comment:
- Same with above, we can have accumulated capacity even without investments, i.e., for existing capacity. So a needs a different set.
- Another thing here is we may have some existing capacity in 2030, but no longer in 2050, then we would create an expression in 2050 that is 0. Is that a problem? I think we should try to not create empty expressions, but here since it is only existing units, I don't know how to avoid this.

Reply:
Latest changes account for the reviewed formulation